dart/lib/src/protocol/device.dart

```diff
@@ -162,6 +162,9 @@ class Device {
       case Orientation.landscape:
         orientation = 'landscape';
         break;
+      case null:
```

Comment:
damn, can we use a `default` then? makes more sense IMO

Reply:
I like exhaustive cases more than default, because the compiler complains if another value gets added to the enum.
Also I could image people wondering why it has a default even though it looks exhaustive. Your call however

Reply:
I agree with your point of view when looking at readability